WooCommerce new documentation (#1299)
mercadopago/devsite-docs: guides/plugins/woocommerce/integration.en.md

# Integration setup

Currently, there are four types of checkout available for WooCommerce. If you want to configure all payment methods offered, you must do the processes individually. Otherwise, choose the one that best suits your business and configure it as follows:

1. Access the WordPress **Panel**.
2. Click on **Plugins > Installed Plugins**.
3. Search for **Mercado Pago payments for WooCommerce** and click on _Configurar_.
4. Select **1 or more checkouts/payment options** you want to offer and click on **Manage** to open the plugin management screen.

On the plugin management screen, fill in the required fields according to your business information. Take into account the information in the sections below: 

## Country of operation

In the *País de operação* field, you must choose the country in which your Mercado Pago account operates. 

## Credentials activation

> In the *Add credentials to "Test Mode" or "Production Mode"* section, you must fill in your **production** and **test** credentials.

Author: ```suggestion
In the **Add credentials to "Test Mode" or "Production Mode"** section, you must fill in your **production** and **test** credentials.
```

The Production credential will enable the store to process actual sales at the end of the initial tests. It is through this credential that the activation is complete.

The Test credential will be necessary at first to carry out tests to ensure the correct functioning of the purchase and payment flow.

To activate the credentials, follow the steps below.

1. Scroll down to the _Credenciais_ section.
2. On **Add credentials to "Test Mode"or "Production Mode"**, choose **Activate Test Mode for Mercado Pago checkouts**. (By keeping this field enabled, your store will be in **test mode**, which allows you to test the plugin before setting the store to production).

> 3. Enter your **test** and **production** credentials in the required fields. If you don't have this information, access the [Credentials](https://www.mercadopago.com.br/developers/en/guides/resources/credentials) documentation and follow the required steps. 

Reviewer: ```suggestion
3. Enter your **test** and **production** credentials in the required fields. If you don't have this information, access the [Credentials](https://www.mercadopago.[FAKER][DOMAIN][URL]/developers/en/guides/resources/credentials) documentation and follow the required steps. 
```

4. When finished filling in, click on _Salvar alterações_.

Set up the business information as shown in the next section once you fill in the credentials. 

## Business information

*Informações de negócio* is required to identify your store. Fill in the fields as below.

1. **Nome da loja:** Enter the name of your store.
2. **Categoria da loja:** Enter the category of your store's products.
3. **Store ID:** Use a number or prefix to identify orders and payments from your store.
4. **Integrator ID:** Enter your *integrator_id* as Mercado Pago partner. If you are not a partner yet, [click here](https://www.mercadopago.com.br/developers/en/developer-program#dev-program-benefits) for more information.

> In _Ajustes avançados_, you can configure options related to saving information to a file for debugging technical issues, as well as configuring [IPN notifications](https://www.mercadopago.com.br/developers/en/guides/notifications/ipn).

Reviewer: ```suggestion
In _Ajustes avançados_, you can configure options related to saving information to a file for debugging technical issues, as well as configuring [IPN notifications](https://www.mercadopago[FAKER][DOMAIN][URL]/developers/en/guides/notifications/ipn).
```

7. **Voltar à loja:** Select whether or not you want the customer to automatically return to your store after completing the payment.
8. **URL de sucesso / URL de pagamento recusado / URL de pagamento pendente:** If you want to build a URL and customize the return page for the 3 statuses informed, just enter them in the requested field.
9. **Modo binário:** Activate when you don't want to leave payments in pending or review status. With binary mode, payments will be accepted or declined automatically.
10. **Cupons de desconto:** Select whether or not you want to offer discount coupons in your store.
11. **Reduzir inventário:** Select **Yes** if you want the product to be taken out of stock during order creation, regardless of whether the payment is approved or not. Otherwise, keep **No** to have the product withdrawn from stock only when payment is approved. 
12. **Desconto por comprar com Mercado Pago:** Set a percentage discount amount for customers who pay with Mercado Pago.

> 13. **Comissão por compra com Mercado Pago:** Set an additional percentage amount that you want to charge as a fee to your customers for paying with Mercado Pago.

Reviewer: ```suggestion
13. **Comissão por compra com Mercado Pago:** Set an additional percentage amount that you want to charge as a fee to your customers for paying with Mercado Pago.
14. **Pagamento com cartões salvos**: Allows customers to buy with their card details saved at Mercado Pago, without having to fill in card details at the store checkout.
```

----[mlb]----

> 14. **Pix:** If you want to offer Pix payments, you need to activate **Custom Checkout** **- Pay with Pix** and follow the steps described on the screen to complete the integration. **(Brazil only)**

Reviewer: ```suggestion
15. **Pix:** If you want to offer Pix payments, you need to activate **Custom Checkout** **- Pay with Pix** and follow the steps described on the screen to complete the integration. **(Brazil only)**
```

> 15. **Vencimento do Pix:** The validity period of the code sent to the customer after placing the order. This will be the period the customer will have to pay for the purchase. **(Brazil only)**

Reviewer: ```suggestion
16. **Vencimento do Pix:** The validity period of the code sent to the customer after placing the order. This will be the period the customer will have to pay for the purchase. **(Brazil only)**
```

------------

> 16. **Pagamento com cartões salvos**: Allows customers to buy with their card details saved at Mercado Pago, without having to fill in card details at the store checkout.

Reviewer: ```suggestion

```

----[mlb]----
> WARNING
>
> Important
>

> >Before configuring Pix as a payment method, we recommend [downloading the latest version](https://br.wordpress.org/plugins/woocommerce-mercadopago/#description) of the Mercado Pago plugin for WooCommerce and [registering your Pix key](https://www.mercadopago.com.br/stop/pix?url=https%3A%2F%2Fwww.mercadopago.com.br%2Fadmin-pix-keys%2Fmy-keys&authentication_mode=required) in Mercado Pago. **(Brazil only)**

Reviewer: ```suggestion
>Before configuring Pix as a payment method, we recommend [downloading the latest version](https://br.wordpress.org/plugins/woocommerce-mercadopago/#description) of the Mercado Pago plugin for WooCommerce and [registering your Pix key](https://www.mercadopago.com.br/stop/pix?url=https%3A%2F%2Fwww.mercadopago.com.br%2Fadmin-pix-keys%2Fmy-keys&authentication_mode=required) in Mercado Pago.
```
